package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "typedoc-default-themes",
   "description": "Default themes for TypeDoc.",
-  "version": "0.4.1",
+  "version": "0.4.2",
```

Comment:
Since the npm script runs on every CI build (since it's not prebuilt, I need to built it myself in Travis/Appveyor) this will need to be a new version (so I can update it in my package.json).